### PR TITLE
fix: quick metrics setup in kind with prometheus

### PR DESCRIPTION
**Pull request by adodon2go**
**What type of PR is this?**
bug

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:
We have a script in examples/metrics to quickly setup a kind cluster with prometheus support.
That can be used together with a running grafana instance to quickly display some graphs.

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andaaron — APPROVED** [review]
lgtm

File: examples/metrics/Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ CONTAINER_RUNTIME := docker
 
 .PHONY: binary-container
 binary-container:
-	${CONTAINER_RUNTIME} build -f build/Dockerfile -t zot-build:latest ../../.
+	${CONTAINER_RUNTIME} build -f Dockerfile -t zot-build:latest ../../.
```
Comment (rchincha) [inline examples/metrics/Makefile:5]: Our Dockerfiles are located under build/ right?

File: examples/metrics/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # ---
 # Stage 1: Install certs, build binary, create default config file
 # ---
-FROM ghcr.io/project-zot/golang:1.19 AS builder
+FROM ghcr.io/project-zot/golang:1.20 AS builder
```
Comment (rchincha) [inline examples/metrics/Dockerfile:4]: Ideally, this should be build-arg'ed out but that also means whoever uses this needs to be aware.

File: examples/metrics/Makefile (hunk 1 above)
Comment (adodon2go) [reply examples/metrics/Makefile:5]: Not anymore. Not sure who/why removed them from there (can be investigated)

Comment (andaaron) [reply examples/metrics/Makefile:5]: Are we discussing https://github.com/project-zot/zot/blob/main/build/Dockerfile? Or others?

Comment (rchincha) [reply examples/metrics/Makefile:5]: https://github.com/project-zot/zot/tree/main/build
^ i still see them?

**Comment by rchincha** [comment]
Also can this be added as a nightly test?

**Review by andaaron — COMMENTED** [review]
This looks good from my side.

**Comment by adodon2go** [comment]
> Also can this be added as a nightly test?

I will add a new CI workflow to address this.

**Comment by andaaron** [comment]
> > Also can this be added as a nightly test?
> 
> I will add a new CI workflow to address this.

Let's track this in https://github.com/project-zot/zot/issues/1930

**Review by andaaron — APPROVED** [review]

**Pull request merged**